metabase/metabase
# Add syncing progress indicators

Epic: https://github.com/metabase/metabase/issues/18675

Changes:
- Adds a snackbar and a special modal for admins who have created a database.
- Adds checks for not yet loaded databases & tables

How to test:
- Admin > Databases
- Copy H2 connection string from the sample database
- Add a new H2 database with the same connection string
- For the first time, you should see both the modal and the snackbar
- The snackbar should show progress and then disappear
- If you add another database, only the snackbar will appear

<img width="272" alt="Screen Shot 2021-12-07 at 8 13 56 pm" src="https://user-images.githubusercontent.com/8542534/145075260-4bcd4172-11c2-41fe-b98a-1c24d2dd7ef8.png">

<img width="274" alt="Screen Shot 2021-12-07 at 8 13 39 pm" src="https://user-images.githubusercontent.com/8542534/145075253-18d53983-d150-4550-aab7-206e5355eb0d.png">
